.github/workflows/nv-mii.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
       - name: Install pytorch
         run: |
-          pip3 install -U --cache-dir $TORCH_CACHE torch
+          pip3 install -U --cache-dir $TORCH_CACHE torch --index-url https://download.pytorch.org/whl/cu118
```

Comment:
I don't know if we have cuda 118 runners, but should we update the runs-on here to be 118 if possible?

Reply:
Any of the CI workflows that we want to use "torch-latest" with will need to be updated to 11.8 - torch 2.1 is only precompiled for CUDA 11.8 and 12.1